vagrant/provisioning/roles/arkcase-app/templates/server.yml

```diff
@@ -43,7 +43,8 @@ wopi.plugin:
     domain: "{{ external_host }}"
     port: 443
 application:
-  rolesToGroups: "{{ roles_to_groups }}"
+  rolesToGroups:
+    {{ roles_to_groups | to_nice_yaml | indent(4) }}
```

Comment:
I need to read up on all the Jinja2 filters... I don't remember seeing to_nice_yaml  or indent before.

Reply:
I did spent 2-3 hours to come up with a way to format this one correctly.